lottie-ios/Classes/AnimatableLayers/LOTLayerView.h

```diff
@@ -13,7 +13,7 @@
 
 @interface LOTLayerView : LOTAnimatableLayer
 
-- (instancetype)initWithModel:(LOTLayer *)model inLayerGroup:(LOTLayerGroup *)layerGroup;
+- (instancetype)initWithModel:(LOTLayer *)model inLayerGroup:(LOTLayerGroup *)layerGroup inBundle:(NSBundle *)bundle;
```

Comment:
can you leave `- (instancetype)initWithModel:(LOTLayer *)model inLayerGroup:(LOTLayerGroup *)layerGroup;` so that the header will maintain legacy support? Just add support for passing in nil bundle and have the shorter method call the longer one internally.